adapters/eth_tx_test.go

```diff
@@ -527,8 +527,6 @@ func TestEthTxAdapter_Perform_NotConnected(t *testing.T) {
 	defer cleanup()
 	store := app.Store
 
-	assert.NoError(t, app.Start())
```

Comment:
Now that we connect immediately instead of 1 second later, `Start` introduces race conditions. Remove where applicable in tests for pending connection.